sdcm/tester.py

```diff
@@ -1807,6 +1807,11 @@ def run_stress_cassandra_thread(
         else:
             timeout = get_timeout_from_stress_cmd(stress_cmd) or self.get_duration(duration)
 
+        if "no-warmup" not in stress_cmd and "stress user" not in stress_cmd:
```

Comment:
I think it's better to use `user profile=`
```suggestion
        if "no-warmup" not in stress_cmd and "user profile=" not in stress_cmd:
```

Reply:
Also we are sure user-profile doesn't do warm-up ?

Reply:
damn, I tested it but didn't notice it was already set. It supports no-warmup.

Reply:
fixed, and I found one more use case where before c-s was some java settings provided - also fixed.